scripts/ui/topological-ui-run.sh

```diff
@@ -0,0 +1,32 @@
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
+
+export BASE_DIR=~/Projects
+
+# binding a dir into a container does not work when enforcing
+echo ${BOLD}Checking if Enforcing${NORMAL}
+getenforce | grep Enforcing && sudo setenforce 0
+
+# start docker if not running
+echo ${BOLD}Checking Docker${NORMAL}
+service docker status | grep inactive && sudo service docker start
```

Comment:
@martinpovolny why is docker needed here?

Reply:
because part of the setup runs in docker